decidim/decidim
# Nullable component setting generates error :comments_max_length

#### :tophat: What? Why?
This PR transforms `:comments_max_length` as mandatory field in all the components to prevent breaking the app. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #10548
- Related to #10551
- Fixes #10560

#### Testing

1. Go to meetings component admin
2. Set the Comments max length to empty string
3. Save component
4. View public page
5. See error
6. To test, we need to repeat steps 2, 3, 4 
7. see the error is gone

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
